src/OpenLoco/Paint/Paint.cpp

```diff
@@ -764,6 +764,12 @@ namespace OpenLoco::Paint
         call(0x0045EA23);
     }
 
+    // 0x0045A60E
+    void PaintSession::drawStringStructs()
```

Comment:
What does this function do?

Reply:
Draws the string structs :p. String structs are basically just used for the floating currency entities.

Reply:
Ah, that wasn't obvious to me. I'm sure it'll make sense once implemented, though.